projects/drivers/xilinx_platform/platform_drivers.h

```diff
@@ -81,141 +74,56 @@ typedef enum i2c_type {
 	XILINX_I2C
 } i2c_type;
 
-typedef struct i2c_init_param {
+typedef struct extra_i2c_init_param {
```

Comment:
why the `extra_` prefix ?
shouldn't `i2c_init_param` from i2c.h be sufficient?

Reply:
if this is xilinx specific, then maybe move this entirely to `platform_drivers.c` and rename to `xil_i2c_init_param`

Reply:
@commodo is right, the header file should be platform agnostic.

Reply:
Question: should I remove the header file completely within this commit? Since, I suppose, we will use the generic header file: https://github.com/analogdevicesinc/no-OS/blob/master/drivers/platform/generic/platform_drivers.h ?

Reply:
I guess we can do this step now.